frame/collective/src/lib.rs

```diff
@@ -425,25 +476,26 @@ decl_module! {
 		///       - 1 storage write `Voting` (codec `O(M)`)
 		///   - 1 event
 		/// # </weight>
-		#[weight = (
+		#[pallet::weight((
 			if *threshold < 2 {
 				T::WeightInfo::propose_execute(
 					*length_bound, // B
 					T::MaxMembers::get(), // M
 				).saturating_add(proposal.get_dispatch_info().weight) // P1
 			} else {
 				T::WeightInfo::propose_proposed(
-					*length_bound, // B
-					T::MaxMembers::get(), // M
-					T::MaxProposals::get(), // P2
+				*length_bound, // B
+				T::MaxMembers::get(), // M
+				T::MaxProposals::get(), // P2
```

Comment:
previous indentation was more sensible
```suggestion
					*length_bound, // B
					T::MaxMembers::get(), // M
					T::MaxProposals::get(), // P2
```